### ISSUE TITLE
Delete style-guide.md

### ISSUE THREAD

**Issue by allanaaa** [omeka/omeka-s-enduser#131]
All of the user manual stylistic & accessibility information should be in the internal Google Doc that already exists for this purpose. If we want a public-facing "How To Edit The Manual" guide, it should be in the readme only and not in style-guide.md. After those two locations are updated and consistent, delete style-guide.md. 

Don't forget to update the readme on the Classic docs too: https://github.com/omeka/classic-enduser
